pensieve/tests/test_experimenter.py

```diff
@@ -157,3 +190,16 @@ def test_started_since(experiment_collection):
     recent = experiment_collection.started_since(dt.datetime(2019, 1, 1, tzinfo=pytz.utc))
     assert isinstance(recent, ExperimentCollection)
     assert len(recent.experiments) > 0
+
+
+def test_end_after(experiment_collection):
+    active_experiments = experiment_collection.end_after(dt.datetime(2019, 12, 1, tzinfo=pytz.utc))
+    assert len(active_experiments.experiments) == 1
+    assert active_experiments.experiments[0].slug == "doh-us-engagement-study-v2"
+
+
+def test_normandy_experiment_slug(experiment_collection):
+    normandy_slugs = list(map(lambda e: e.normandy_slug, experiment_collection.experiments))
+    assert "addon-activity-stream-search-topsites-release-69-1576277" in normandy_slugs
+    assert None in normandy_slugs
+    assert "pref-doh-us-engagement-s…lease-69-71-bug-1590831" in normandy_slugs
```

Comment:
That ellipsis literal didn't come out of the Experimenter API did it?

Reply:
oops, no. Copy-paste error of the collapsed JSON string

Reply:
phew. I would have been disappointed. :D